apps/dapp-console-api/src/models/addresses.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  index,
+  jsonb,
+  pgTable,
+  timestamp,
+  uniqueIndex,
+  uuid,
+  varchar,
+} from 'drizzle-orm/pg-core'
+import type { Address } from 'viem'
+
+import { entities } from './entities'
+
+enum AddressState {
+  ACTIVE = 'active',
+  DISABLED = 'disabled',
+  SANCTIONED = 'sanctioned',
+}
+
+type AddressVerification = {}
+
+export const addresses = pgTable(
+  'addresses',
+  {
+    createdAt: timestamp('created_at', { withTimezone: true })
+      .defaultNow()
+      .notNull(),
+    updatedAt: timestamp('updated_at', { withTimezone: true })
+      .defaultNow()
+      .notNull(),
+    id: uuid('id').defaultRandom().primaryKey(),
+    entityId: uuid('entity_id').references(() => entities.id),
```

Comment:
realized this after looking at the other PR, which should make this not null we don't want to be able to save anything to the db without an entity ref

Reply:
good catch!